utils/link_issues_CHANGELOG

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+in=CHANGELOG.md
+
+# Replace them with Markdown references
+sed -i -e 's/(\(#[0-9]\+\))/([\1])/g' "$in"
```

Comment:
as recently discovered, for "proper" original markdown, so we could convert using pandoc into .rst, should be links such as `[#XXX][]` not just `[#XXX]`.  I will push a fixup directly into this branch